src/utils/constants.ts

```diff
@@ -273,10 +274,11 @@ export const FORMATTED_BH_NAMES: FormattedBHNames = {
   hunterV2: 'Bounty Hunter - Hunter'
 };
 
-export const FORMATTED_LMS = 'Last Man Standing';
-export const FORMATTED_PVP_ARENA = 'PvP Arena';
+export const FORMATTED_LMS = 'LMS - Rank';
+export const FORMATTED_PVP_ARENA = 'PvP Arena - Rank';
 export const FORMATTED_SOUL_WARS = 'Soul Wars Zeal';
 export const FORMATTED_LEAGUE_POINTS = 'League Points';
-export const FORMATTED_RIFTS_CLOSED = 'Rifts Closed';
+export const FORMATTED_RIFTS_CLOSED = 'Rifts closed';
```

Comment:
I wonder if we consider this a breaking change. On one hand, they are just display names that hopefully users are not relying on the value being static. But on the other hand they are now being treated as unique identifiers so updating them after this PR lands could be breaking. 🤔